### DISCUSSION TITLE
Include Wikipedia information in API result

### DISCUSSION THREAD

**Discussion by georgbachmann** [komoot/photon#604] [Q&A]
I did a little bit of research but didn't find useful informations. OSM has wikipedia tags. Seems like you are also using it for e.g. relevance-scoring if I understood some of the issues related to wikipedia correct. Also there seems to be a flag `--import-wikipedia-articles` for the importer.

So clearly wikipedia is somehow used. So maybe I am just too blind to see how I can export that information when doing a search-query? My goal would be to be able to provide my user with more informations on a place if then which.

I am only using the sample api you guys provided (thank you!!!). Is it a limitation of that one? Would importing my own dataset solve the issue?

**Comment by lonvia** [comment]
We don't export any of the extra tags that Nominatim keeps by default. So you cannot get information about wikipedia links from the standard Photon exports that we provide.

However, you can add any additional information you need in your own exports of the Photon database using the handy new [--extra-tags feature](https://github.com/komoot/photon/pull/576). Simply run the export with `--extratags wikipedia` and the importer will include the information in the Photon database and automatically return it under `properties->extra->wikipedia` for all places that have the information. Needs the latest release of Photon.
